src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -132,8 +117,40 @@ else if (!settingsDir.isDirectory())
     }
 
     /**
+     * Parse the options for the javadoc command line. The first item is the option name, and the
+     * following the number of items returned by {@link #optionLength(String)} for that option.
+     *
+     * Implementations should override this method and call it to parse the required options.
+     * In addition, to support custom options {@link #optionLength(String)} may be implemented.
```

Comment:
Change this to:

Custom Barclay doclets that want to have custom command line options should override this method, and also provide an implementation of the static method {@link #optionLength}). Both methods should handle the custom options, and then delegate back to the base class methods defined here to allow default handling of builtin options.